### PR TITLE
Updates translation to Swedish (Engine)

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

This PR updates the translation (PO file) for Swedish.

## How to test this PR

Change to Swedish and verify that translation comes out as expected for updated msgids and msgstrs.

### CODE REVIEW

File: share/sv.po [hunk 1]
```diff
@@ -630,6 +630,74 @@ msgstr ""
...
+#. DNSSEC:DS01_DS_ALGO_NOT_DS
+#, perl-brace-format
+msgid ""
+"DS record for zone {domain} with keytag {keytag} was created by digest "
+"algorithm {ds_algo_num} ({ds_algo_mnemo}) which is not meant for DS. Fetched "
+"from the nameservers with IP addresses \"{ns_ip_list}\"."
+msgstr ""
+"DS-posten för zonen {domain} med \"key tag\" {keytag} skapades med "
+"hashfunktionen (\"digest algorithm\") {ds_algo_num} ({ds_algo_mnemo}) som "
+"inte är avsedd för DS-post. Hämtad från namnservrarna med IP-adresserna "
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:673]: Either "DS-post", "en DS-post" or "DS-poster" could be used in such a context depending on style. I'll update with the indefinite article.

File: share/sv.po [hunk 2]
```diff
@@ -630,6 +630,74 @@ msgstr ""
...
+
+#. DNSSEC:DS01_DS_ALGO_RESERVED
+#, perl-brace-format
+msgid ""
+"DS record for zone {domain} with keytag {keytag} was created with an "
+"unassigned digest algorithm (algorithm number {ds_algo_num}). Fetched from "
+"the nameservers with IP addresses \"{ns_ip_list}\"."
+msgstr ""
+"DS-posten för zonen {domain} med \"key tag\" {keytag} skapades med en "
+"hashfunktion (\"digest algorithm\") med algoritmnummer {ds_algo_num}. "
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:684]: Actually, there is none. That is why I use the indefinite article, not in definite form.

```po
msgid "DS record for zone {domain} with keytag {keytag} was created with an "
"unassigned digest algorithm (algorithm number {ds_algo_num}). Fetched from "
"the nameservers with IP addresses \"{ns_ip_list}\"."

msgstr "DS-posten för zonen {domain} med \"key tag\" {keytag} skapades med en "
"hashfunktion (\"digest algorithm\") med algoritmnummer {ds_algo_num}. "
"Algoritmnumret är inte tilldelat någon hashfunktion. Hämtad från "
"namnservrarna med IP-adresserna \"{ns_ip_list}\"."
```

File: share/sv.po [hunk 3]
```diff
@@ -630,6 +630,74 @@ msgstr ""
...
+#. DNSSEC:DS01_DIGEST_NOT_SUPPORTED_BY_ZM
+#, perl-brace-format
+msgid ""
+"DS record for zone {domain} with keytag {keytag} was created by digest "
+"algorithm {ds_algo_num} ({ds_algo_mnemo}) which cannot be validated by this "
+"installation of Zonemaster. Fetched from the nameservers with IP addresses "
+"\"{ns_ip_list}\"."
+msgstr ""
+"DS-posten för zonen {domain} med \"key tag\" {keytag} skapades med "
+"hashfunktionen (\"digest algorithm\") {ds_algo_num} ({ds_algo_mnemo}) som "
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:642]: Every time the expression is 'hashfunktionen ("digest algorithm")'. That should be clear enough. Isn't it?

File: share/sv.po [hunk 4]
```diff
@@ -1986,6 +1976,33 @@ msgstr ""
...
+"Inget svar på en fråga med EDNS version 1. Hämtad från namnservrarna med IP-"
+"adresserna \"{ns_ip_list}\"."
+
+#. N10_UNEXPECTED_RCODE
+#, perl-brace-format
+msgid ""
+"Erroneous RCODE (\"{rcode}\") in response to an EDNS version 1 query. "
+"Fetched from the nameservers with IP addresses {ns_ip_list}"
+msgstr ""
+"Felaktig RCODE (\"{rcode}\") i svaret på en fråga med EDNS version 1. Hämtad "
```
[the remark this answers is not in the document]
Comment (matsduf) [reply share/sv.po:1994]: Corrected.

File: share/sv.po (hunk 3 above)
[the remark this answers is not in the document]
Comment (mattias-p) [reply share/sv.po:642]: Sure, if our aim is set on good enough. I meant to give a suggestion for how to make it slightly better. But you don't have to.

**Review by mattias-p — APPROVED** [review]

**Pull request merged**